python/ray/rllib/optimizers/evaluator.py

```diff
@@ -62,6 +74,49 @@ def set_weights(self, weights):
 
         raise NotImplementedError
 
+    def merge_filters(self, obs_filter=None, rew_filter=None):
```

Comment:
Let's remove these methods from this class. We can move them into a separate class that manages the filter state.